strings/c-or-cpp/Permutation-of-String.c

```diff
@@ -55,11 +55,13 @@ void swap(char *a,char *b)
     *b=temp;
 	//swapping done
 }
-// Example:
-//Input: abc
-//Output: 1(abc)
-//        2(acb)
-//        3(bac)
-//        4(bca)
-//        5(cba)
-//        6(cab)
+/* Example:
```

Comment:
did the multi comment looks good or not

Reply:
yes, multiline comments looks visually better.